### DISCUSSION TITLE
Add to get-RscVsphereVm cmdlet

### DISCUSSION THREAD

**Discussion by Freakwncy** [rubrikinc/rubrik-powershell-sdk#11] [Ideas]
get-RscVsphereVm should have Protection date as well as LastSuccessful backup added. And cluster name should not be nested.

PS C:\Users\rhobbs> get-RscVsphereVm -Name ustry1mety0187a

Id             : 254c36db-2a58-573a-9dc7-d11a8b6b0b19
Name           : USTRY1METY0187A
GuestOsType    : LINUX
ProtectionDate :
Cluster        : RubrikSecurityCloud.Types.Cluster

Similar to this:
PS C:\Users\rhobbs> Invoke-Rsc "query GetVsphereVmList{vSphereVmNewConnection{nodes{id name protectionDate guestOsType cluster{name}}}}" | select -ExpandProperty Nodes | where-object -Property Name -eq "ARBUEFLSP01"       

Id             : 07832a5e-3018-58c6-b2da-0d148c49cc8c
Name           : ARBUEFLSP01
GuestOsType    : WINDOWS
ProtectionDate : 11/18/2022 5:28:24 PM
Cluster        : RubrikSecurityCloud.Types.Cluster

**Comment by Freakwncy** [comment]
The #1 question we get from most people is "Is my server being backup? and if so what's the last backup?"

**Comment by guirava** [comment]
thank you for your feedback, this cmdlet was deprecated in favor of `Get-RscVmwareVm`.

thank you for your suggestion to add protection date and  LastSuccessful backup , they will be added to release 1.1

please re-open discussion if you have more questions or run into any issues.
